lib/pbench/server/api/resources/alembic/script.py.mako

```diff
@@ -0,0 +1,24 @@
+"""${message}
```

Comment:
What is a `.mako` file?

Reply:
@portante  This is an auto generated file after we init the alembic migrations by `alembic init alembic`. This is a [Mako](http://www.makotemplates.org/) template file which is used to generate new migration scripts. Whatever is here is used to generate new files within `versions/`. This is scriptable so that the structure of each migration file can be controlled, including standard imports to be within each, as well as changes to the structure of the upgrade() and downgrade() functions.

An example migration version file `<revision_id>_<commit_message>.py` can look like following, and if you look at the upgrade and downgrade function thats what we would use to switch between db versions. and what gets written inside this migration file is controlled by the `.mako` file 
```
"""second commit

Revision ID: b2d3c7151d72
Revises: 781e91f9eaf1
Create Date: 2020-12-03 13:17:02.887022

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = 'b2d3c7151d72'
down_revision = '781e91f9eaf1'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('users', sa.Column('test_column', sa.String(length=255), nullable=True))
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_column('users', 'test_column')
    # ### end Alembic commands ###
```